server/views/scss_style.scss

```diff
@@ -0,0 +1,163 @@
+$cell-color: #fdfdfd;
+$background-color: #f3f3f5;
+
+@mixin inputsize() {
+  text-align: center;
+  font-size: 30px;
+}
+
+body{
+  background-color: $background-color;
+}
+.head{
+  display: flex;
+  width: 100%;
+  height: 4vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: $background-color;
+  z-index: 1;
+}
+.uploadFile{
+  flex: 1 50px;
+}
+.cellInputForm{
+  flex: 5 100px;
+  background-color: $cell-color;
+}
+.myFileInput{
+  width: 100%;
+  height: 100%;
+}
+
+.newFile{
+  flex: 1 50px;
+}
+
+.letterRow{
+  display: flex;
+  margin-left: 2vh;
+  margin-top: 4vh;
+  top: 4vh;
+  position: sticky;
+}
+
+.letter{
+  background-color: $background-color;
+  width: 150px;
+  text-align: center;
+}
+
+.numberRow{
+  position: sticky;
+  left: 0vh;
+  background-color: $background-color; 
+  text-align: center;
+  padding-top: 2vh;
+}
+
+.number{
+  width: 2vh;
+  padding-bottom: 10px;
```

Comment:
use more variables for sizes
they are reused several times

Reply:
done